README.md

```diff
@@ -1,3 +1,145 @@
-# self-image-experiments 
-[![Build Status](https://travis-ci.com/quanganhhoang/sie-firebase.svg?branch=master)](https://travis-ci.com/quanganhhoang/sie-firebase)
```

Comment:
could we please keep this line for build status? It actually shows the status of our CI/CD

Reply:
Sure thing!